src/Symfony/Component/EventDispatcher/EventSubscriberInterface.php

```diff
@@ -43,7 +43,7 @@ interface EventSubscriberInterface
      * The code must not depend on runtime state as it will only be called at compile time.
      * All logic depending on runtime state must be put into the individual methods handling the events.
      *
-     * @return array<string, mixed> The event names to listen to
+     * @return array<string, mixed>
```

Comment:
Added. but I'm wondering how useful such descriptions are. Looks quite cryptic.

Reply:
It's by definition a win for static analysers to have the most precise type information possible. I agree that it's no longer that human readable (although it can be helpful as I always forget if it's `['eventName', prio]` or the other way around)